[2024-03-20T03:37:17.102+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example1.t4 scheduled__2024-03-19T00:00:00+00:00 [queued]>
[2024-03-20T03:37:17.106+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example1.t4 scheduled__2024-03-19T00:00:00+00:00 [queued]>
[2024-03-20T03:37:17.107+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-20T03:37:17.112+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): t4> on 2024-03-19 00:00:00+00:00
[2024-03-20T03:37:17.114+0000] {standard_task_runner.py:60} INFO - Started process 263 to run task
[2024-03-20T03:37:17.116+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'example1', 't4', 'scheduled__2024-03-19T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/example1.py', '--cfg-path', '/tmp/tmp9u9hzpm2']
[2024-03-20T03:37:17.117+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask t4
[2024-03-20T03:37:17.124+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-20T03:37:17.141+0000] {task_command.py:423} INFO - Running <TaskInstance: example1.t4 scheduled__2024-03-19T00:00:00+00:00 [running]> on host d3305dc92eb8
[2024-03-20T03:37:17.172+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example1' AIRFLOW_CTX_TASK_ID='t4' AIRFLOW_CTX_EXECUTION_DATE='2024-03-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-19T00:00:00+00:00'
[2024-03-20T03:37:17.173+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-20T03:37:17.173+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo task 4']
[2024-03-20T03:37:17.179+0000] {subprocess.py:86} INFO - Output:
[2024-03-20T03:37:17.180+0000] {subprocess.py:93} INFO - task 4
[2024-03-20T03:37:17.180+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-20T03:37:17.189+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=example1, task_id=t4, execution_date=20240319T000000, start_date=20240320T033717, end_date=20240320T033717
[2024-03-20T03:37:17.222+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-20T03:37:17.229+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
